plugins/woocommerce/src/Caches/OrderCacheController.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Automattic\WooCommerce\Caches;
+
+use Automattic\WooCommerce\Internal\Features\FeaturesController;
+use Automattic\WooCommerce\Internal\Traits\AccessiblePrivateMethods;
+
+/**
+ * A class to control the usage of the orders cache.
+ */
+class OrderCacheController {
+
+	use AccessiblePrivateMethods;
+
+	const FEATURE_NAME = 'orders_cache';
```

Comment:
We can be explicit about the visibility level:

```suggestion
	public const FEATURE_NAME = 'orders_cache';
```

Reply:
Note that I have merged the feature flag with HPOS feature.